mobile/library/java/org/chromium/net/impl/NativeCronvoyEngineBuilderImpl.java

```diff
@@ -52,7 +52,7 @@ public class NativeCronvoyEngineBuilderImpl extends CronvoyEngineBuilderImpl {
   private final boolean mEnableDnsFilterUnroutableFamilies = true;
   private boolean mUseCares = false;
   private final List<Pair<String, Integer>> mCaresFallbackResolvers = new ArrayList<>();
-  private boolean mUseGro = false;
+  private boolean mUseGro = true;
```

Comment:
Would it make more sense to change the default value of the flag in runtime_guards.cc and remove the config knobs?
/wait-any

Reply:
We need to make the RUNTIME_GUARD default true for sure, but I think this change is also needed until we deprecate the RUNTIME_GUARD.

Reply:
After all, shall we wait for the experiment result before submit this change?

Reply:
which experiment are you referring to?

Reply:
We are doing an Android experiment with preferring GRO this week, and I assume we can know the result early next week.

Reply:
sure, we can wait till then.  Since I have the PR up, I'll prepare the changes anyway.

RE: server side Envoy, are we comfortable with flipping the runtime guard to true by default?

Reply:
> RE: server side Envoy, are we comfortable with flipping the runtime guard to true by default?

Yes, that's the expected procedure for any default-true envoy flags.